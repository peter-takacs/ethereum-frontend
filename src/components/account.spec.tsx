import * as React from 'react';
import AccountDisplay from './Account';
import * as renderer from 'react-test-renderer';

test('Renders with correct account', () => {
    const component = renderer.create(
        <AccountDisplay address='0xDEADBEEF' />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})