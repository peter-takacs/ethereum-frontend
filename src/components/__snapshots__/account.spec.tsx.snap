// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders with correct account 1`] = `
<div>
  Current account is: 
  0xDEADBEEF
</div>
`;
